### PR TITLE
refactor: Changed opcode costs per EIP-1884

### PR DESCRIPTION
**Pull request by fubuloubu**
fixes: #1653

### What I did
Changed a few opcodes

### How to verify it
Tests still pass... interestingly enough (our gas estimation features seem not to do anything?)

### Description for the changelog
Add support for Istanbul opcode gas cost changes

### Cute Animal Picture
lazy panda

### CODE REVIEW

File: vyper/opcodes.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
     'BYTE': [0x1a, 2, 1, 3],
     'SHA3': [0x20, 2, 1, 30],
     'ADDRESS': [0x30, 0, 1, 2],
-    'BALANCE': [0x31, 1, 1, 400],
+    'BALANCE': [0x31, 1, 1, 700],
```
Comment (charles-cooper) [inline vyper/opcodes.py]: Can you add a comment to the effect of ISTANBUL RULESET

File: vyper/opcodes.py [hunk 2]
```diff
@@ -52,7 +52,7 @@
     'MLOAD': [0x51, 1, 1, 3],
     'MSTORE': [0x52, 2, 0, 3],
     'MSTORE8': [0x53, 2, 0, 3],
-    'SLOAD': [0x54, 1, 1, 200],
+    'SLOAD': [0x54, 1, 1, 800],
```
Comment (charles-cooper) [inline vyper/opcodes.py]: Can you add a comment to the effect of ISTANBUL RULESET

File: vyper/opcodes.py (hunk 1 above)
Comment (fubuloubu) [reply vyper/opcodes.py]:
```suggestion
    'BALANCE': [0x31, 1, 1, 700],  # Under Istanbul Ruleset
```

File: vyper/opcodes.py (hunk 2 above)
Comment (fubuloubu) [reply vyper/opcodes.py]:
```suggestion
    'SLOAD': [0x54, 1, 1, 800],  # Under Istanbul Ruleset
```

File: vyper/opcodes.py [hunk 3]
```diff
@@ -30,7 +30,7 @@
     'BYTE': [0x1a, 2, 1, 3],
     'SHA3': [0x20, 2, 1, 30],
     'ADDRESS': [0x30, 0, 1, 2],
-    'BALANCE': [0x31, 1, 1, 400],
+    'BALANCE': [0x31, 1, 1, 700],  # Under Istanbul Ruleset
```
Comment (fubuloubu) [inline vyper/opcodes.py]:
```suggestion
    'BALANCE': [0x31, 1, 1, 700],  # Updated Istanbul Ruleset
```

File: vyper/opcodes.py [hunk 4]
```diff
@@ -52,7 +52,7 @@
     'MLOAD': [0x51, 1, 1, 3],
     'MSTORE': [0x52, 2, 0, 3],
     'MSTORE8': [0x53, 2, 0, 3],
-    'SLOAD': [0x54, 1, 1, 200],
+    'SLOAD': [0x54, 1, 1, 800],  # Under Istanbul Ruleset
```
Comment (fubuloubu) [inline vyper/opcodes.py]:
```suggestion
    'SLOAD': [0x54, 1, 1, 800],  # Updated Istanbul Ruleset
```

**Pull request merged**